CHANGELOG.md

```diff
@@ -1,3 +1,239 @@
+<!--
+# YY.0M (YYYY-0M-0D)
+
+Breaking changes:
+
+Features:
+
+Commands:
+
+Usability improvements:
+
+Fixes:
+
+Themes:
+
+New languages:
+
+Updated languages and queries:
+
+Packaging:
+-->
+
+# 25.01 (2025-01-01)
+
+`git log` says there are 164 contributors since 24.07
+
+checkpoint <https://github.com/helix-editor/helix/compare/a5a7cff311e909ecab41f840742a5309634aeec3...master>
+
+<!-- Breaking changes: -->
+
+Features:
+
+* Big refactor for `Picker`s (#9647, #11209, #11216, #11211, #11343, #11406)
+    * Use a table layout and allow filtering by column
+    * Reimplement `global_search` to allow changing the query dynamically
+* Add an alternative "inline" display for LSP diagnostics (#6417, #11815)
+* Support defining keybindings as macros (#4709)
+* Continue line comments in `o`/`O` and on `<ret>` in insert mode (#10996, #12213, #12215)
```

Comment:
Wouldn't this also count as a breaking change?

Reply:
https://github.com/helix-editor/helix/pull/12347#discussion_r1898912515